gateway/gateway.go

```diff
@@ -39,18 +41,17 @@ func New(cfgs ...ConfigFunc) *Client {
 
 	c.APIClient.common.backend.(*defaultBackend).client = &c.client
 
-	for _, cfg := range cfgs {
-		cfg(c)
-	}
-
-	ForURLPattern(manifold.DefaultURLPattern)(c)
+	WithURLEndpoint(baseGatewayURL)(c)
```

Comment:
I'd keep using `ForURLPattern` for some sort of consistency with the main package.